conf/php-fpm.conf

```diff
@@ -28,7 +28,7 @@
 ;                            specific port;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = /var/run/php/php7.0-fpm-__NAMETOCHANGE__.sock
+listen = /run/php/fpm-__NAMETOCHANGE__.sock
```

Comment:
I believe it's better if we stick to the previous place / format because this is kind of a de-facto standard among apps and it helps when yunohost gotta automagically patch stuff..